community/CCLAs/CCLA Instructions.md

```diff
@@ -0,0 +1,8 @@
+## Instructions for handling Corporate Contributor License Agreements (CCLAs)
```

Comment:
Any reason not to add these instructions to CONTRIBUTING.md or to README.md? Otherwise it's not easy for people to find.